docs/series/LAMP-server.md

```diff
@@ -0,0 +1,241 @@
+    ```
+
+* **Create a configuration file in `/etc/httpd/conf.d/` named almasite.conf:**
+    ```sh
+    <VirtualHost *:80>
+        # The domain
+        ServerName almasite.com
+        # The domains alias
+        ServerAlias www.almasite.com
+        # Full path to the website files
+        DocumentRoot /home/almasite/public_html
+        # Full path to the error log file
+        ErrorLog /home/almasite/logs/error.log
+        # A custom log to record requests to the website
+        CustomLog /home/almasite/logs/access.log combined
+        # Apply directives only on a given directory
+        <Directory /home/almasite/public_html>
+            # Enables the server to follow symbolic links within directories
+            Options Indexes FollowSymLinks
+            # Permits the use of .htaccess files to override server configuration settings
+            AllowOverride All
+            # Allow access to the specified directory and its contents to all users
+            Require all granted
+            # Specifies default file for directory access
+            DirectoryIndex index.php
+        </Directory>
+        # Route requests based on patterns
+        ProxyPassMatch ^/(.*\.php(/.*)?)$ fcgi://127.0.0.1:9000/home/almasite/public_html/
+    </VirtualHost>
+    ```
+    Notice the comments explaining each directive.
+
+* **Add a php-fpm pool**
+    ```sh
+    cp /etc/php-fpm.d/www.conf /etc/php-fpm.d/almasite.conf
+    ```
+
+    Open the newly created file with your preferred text editor and change the following:
+    - **\[www\]** - with the pools name (.e.g almasite)
+    - **user = apache** - your user (.i.e almasite)
+    - **group = apache** - your group (.i.e almasite)
+    - **listen = /run/php-fpm/www.soc** - your listening method, in this case we use port 9000 (listen = 9000)
+
+* **Add the websites files: (or file in this case ;))**
+    ```sh
+    touch /home/almasite/public_html/index.php
+    ```
+
+    Open your preferred text editor and add the following:
+    ```html
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <title>Hello world!</title>
+        </head>
+        <body>
+            <?php $now = date("H:i:s", time()); echo "<h1>Hello world!<br>This message is sent from PHP, the time now is $now</h1>" ?>
+        </body>
+    </html>
+    ```
+
+* **Manipulate your hosts file to view your newly created website:**
+
+    ::: tip
+    While I cant tell you your **IP** address, I can tell you that the domain is almasite.com and the alias is www.almasite.com. (That is if you followed my guide exactly)
+
+    ```sh
+    YOUR-IP-ADDRESS www.almasite.com almasite.com
+    ```
+    :::
+
+    <br>
+
+    *<ins>Linux</ins>:*
+
+    Edit your hosts file using vim:
+    ```sh
+    sudo vim /etc/hosts
+    ```
+
+    Then press `i` to enter insert mode and add the following line **end** of the file:
+
+    ```sh
+    [IP ADDRESS] [DOMAIN]
+    ```
+
+    <br>
+
+    *<ins>Windows</ins>:*
```

Comment:
These HTML tags, too

Reply:
Same deal  [here](https://github.com/AlmaLinux/wiki/pull/404#discussion_r1597421430), thanks for the feedback.